aipl/ops/llm.py

```diff
@@ -75,81 +44,32 @@ def op_llm_mock(aipl, v:str, **kwargs) -> str:
 def route_llm_query(aipl, v:str, **kwargs) -> str:
     'Send chat messages to `model` (default: gpt-3.5-turbo).  Lines beginning with @@@s or @@@a are sent as system or assistant messages respectively (default user).  Passes all named args directly to API.'
     model = kwargs.get('model')
-    if model is None:
-        kwargs['model'] = 'gpt-3.5-turbo'
-        return completion_openai(aipl, v, **kwargs)
-    if model in gooseai_models:
-        return completion_gooseai(aipl, v, **kwargs)
-    elif model in openai_pricing:
-        return completion_openai(aipl, v, **kwargs)
+    client_str = kwargs.get('client')
+    if client_str is None:
+        if os.environ['LLM_CLIENT_ENDPOINT']:
```

Comment:
Check if it is there!

Reply:
`AIPL Error (line 50 !llm): 'LLM_CLIENT_ENDPOINT'`

Reply:
should be fixed! test again?